nw/assets/themes/gui/solarized_dark/theme.conf

```diff
@@ -0,0 +1,29 @@
+[Main]
+name       = Solarized Dark
+author     = nullbasis
+credit     = Ethan Schoonover
+url        = https://ethanschoonover.com/solarized/
+license    = MIT
+licenseurl = https://github.com/altercation/solarized/blob/master/LICENSE
+
+[Palette]
+window          =   0,  43,  54
+windowtext      = 253, 246, 227
+base            =   7,  54,  66
+alternatebase   =  67,  67,  67
+text            = 253, 246, 227
+tooltipbase     = 133, 153,   0
+tooltiptext     =   0,  43,  54
+button          =   7,  54,  66
+buttontext      = 253, 246, 227
+brighttext      = 253, 246, 227
+highlight       =  42, 161, 152
+highlightedtext =   0,  43,  54
+link            =  38, 139, 210
+linkvisited     =  38, 139, 210
+
+[GUI]
+treewordcount   = 253, 246, 227
```

Comment:
You can just delete the `treewordcount` entries. They are no longer in use.

Reply:
Oops, I missed this note!